Robust.Shared/Map/Tile.cs

```diff
@@ -120,7 +117,12 @@ public override int GetHashCode()
     }
 }
 
-public enum TileRenderFlag : byte
+public enum TileFlag : byte
 {
-
+    None = 0,
+    Full = 1 << 0,
+    BottomLeft = 1 << 1,
+    BottomRight = 1 << 2,
+    TopRight = 1 << 3,
+    TopLeft = 1 << 4,
```

Comment:
Should these not be two bitfields?
```suggestion
    BottomLeft = 0 << 0, // 2-bit direction
    BottomRight = 1 << 0,
    TopRight = 2 << 0,
    TopLeft = 3 << 0,
    Full = 0 << 2, // 1 bit type.
    Diagonal = 1 << 2,
```

Reply:
Re-opening this, because honestly I think full support for directional tiles is a better thing (if you really don't want to code it, i will, but i don't want to have to be the first person to figure out map format conversion)